Added Community-to-RST #223 (#237)
mautic/mautic-community-handbook: docs/teams/community_team/meetup_guidelines.rst

Guidance for establishing official Mautic meetups, groups & events
##################################################################

> The Community Team agreed on 12/12/2019 the following guidelines

Reviewer: ```suggestion
The Community Team agreed on December 12, 2019 the following guidelines
```

3. Membership in the local meetup group is open to all who wish to join, regardless of ability, skill, financial status or any other criteria.
4. Meetups are volunteer-run with volunteer speakers. In cases where a modest attendance fee might be necessary, this fee should only cover the costs of the meetup and should not be used to pay speakers or organizers.
5. Meetup groups allow events to be organized by any reliable/trusted member of the community.
6. Meetups are welcoming places where everyone works to foster an accepting environment which is free of discrimination, incitement to violence, promotion of hate, and general jerk-like behavior.

Official groups have a Meetup group created on the :xref:`Community Portal` and a category created in :xref:`Meetups Forum Category` on the forums once the following criteria are met: 

1. A thread on the Mautic Community Forums in the :xref:`International Forums` category indicating at least two people interested in joining the meetup
2. A clear region or town for the meetup to be based in, not covered by an existing meetup
3. Two group organisers identified from within the Mautic community, preferably not from the same company

> 4. A location confirmed for meeting which is an office or community space (i.e. not a bar/pub)

Reviewer: ```suggestion
4. A location confirmed for meeting which is an office or community space (not a bar/pub)
```

5. A date for the launch event (ideally at least four weeks ahead to give time for organising, promoting and getting people to attend)

> 6. A proposed time and day each month (e.g. from 6pm on the first Friday of each month) for regular events

Reviewer: ```suggestion
6. A proposed time and day each month (for example from 6 PM on the first Friday of each month) for regular events
```

> Guidance for establishing official Mautic Meetups, Groups & Events

Reviewer: ```suggestion
Guidance for establishing official Mautic meetups, groups & events
```

Vale fix

##################################################################

> The Community Team agreed on 12th December 2019 the following guidelines

Reviewer: ```suggestion
The Community Team agreed on 12/12/2019 the following guidelines
```

for establishing official Mautic groups within the Community.

> All meetups, groups and events agree to abide by the Mautic :xref:`CoC` at all times.

Reviewer: ```suggestion
All meetups, groups, and events agree to abide by the Mautic :xref:`CoC` at all times.
```
Vale fix

> Mautic Meetups

Reviewer: ```suggestion
Mautic meetups
```

**************

> These are the traditional kind of in-person meetups which are based in a

Reviewer: ```suggestion
These are the traditional kind of in-person meetups, based in a
```

> region or a town. They are **not** to be created on a country-wide

Reviewer: ```suggestion
region or a town. They're **not** created on a country-wide
```

basis.

General guidelines for official Mautic Meetups
**********************************************

1. Mautic Meetups are for the benefit of the Mautic community as a whole, not specific businesses or individuals.
2. All actions taken as an event organizer are with the best interest of the community in mind.
3. Membership in the local meetup group is open to all who wish to join, regardless of ability, skill, financial status or any other criteria.
4. Meetups are volunteer-run with volunteer speakers. In cases where a modest attendance fee might be necessary, this fee should only cover the costs of the meetup and should not be used to pay speakers or organizers.
5. Meetup groups allow events to be organized by any reliable/trusted member of the community.
6. Meetups are welcoming places where everyone works to foster an accepting environment which is free of discrimination, incitement to violence, promotion of hate, and general jerk-like behavior.

> Official groups will have a Meetup group created on the :xref:`Community Portal` and a category created in :xref:`Meetups Forum Category` on the forums once the following criteria are met: 

Reviewer: ```suggestion
Official groups have a Meetup group created on the :xref:`Community Portal` and a category created in :xref:`Meetups Forum Category` on the forums once the following criteria are met: 
```
